agent/agent_installer/MacOS/iOS/installer.sh

```diff
@@ -15,6 +15,22 @@ echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> "$userHome/.zprofile"
 eval "$(/opt/homebrew/bin/brew shellenv)"
 brew install carthage
 brew install node
+brew install awk
+brew install android-platform-tools
+
+# trace adb path and set ANDROID_HOME as env var
+adb_path=`which adb`
+# example value of this adb_path here: 
+# lrwxr-xr-x  1 bsp  admin  71 Mar 10 18:47 /opt/homebrew/bin/adb -> /opt/homebrew/Caskroom/android-platform-tools/34.0.0/platform-tools/adb
+adb_path=`ls -l $adb_path`
+echo $adb_path > temp.txt
+adb_path=`awk -F '-> ' '{print $2}' temp.txt`
+echo $adb_path > temp.txt
+export ANDROID_HOME=`awk -F '/platform-tools/' '{print $1}' temp.txt`
+echo $ANDROID_HOME
+echo "\nexport ANDROID_HOME=$ANDROID_HOME" >> export_android_home.sh
```

Comment:
nit: Should we remove the temp.txt and export_android_home.sh after the env set up.

Reply:
I will remove temp.txt and keep export_android_home.sh for debugging.